site/docs/docker-extension.md

```diff
@@ -31,47 +32,77 @@ The Livecycle Docker extension embeds a standalone version of the Preevy CLI, wh
    Private environments require adding team members to your organization, and upon access, your team members will be prompted to authenticate.
    
-   ![Livecycle Docker Extension](docker-ext-5.png)
+   ![Livecycle Docker Extension](https://github.com/livecycle/preevy/assets/51878265/cc2d9c8f-35cd-4d71-a61b-0a4041786bec)
 
 5. **Accessing Livecycle dashboard**
 
    You can also access the Livecycle dashboard to see the logs and debug your application. Click on the "Open Link" button to open the Livecycle dashboard. On the dashboard, you can see all the running applications and services.
    The Livecycle dashboard requires authentication and organization membership, similar to private environments/services 
 
-   ![Livecycle Docker Extension](docker-ext-6.png)
+   ![Livecycle Docker Extension](https://github.com/livecycle/preevy/assets/51878265/c94b28d6-debc-471b-9621-82c73dbc79fe)
 
 6. **Debugging, inspecting, and logging**
 
    Once you have opened the Livecycle dashboard, you can see all the environments/apps that are running. Click on the name of the environment for which you want to see the logs, terminal, etc. You can view the logs, terminal, and container inspection for each service.
 
-   ![Livecycle Docker Extension](docker-ext-7.png)
+   ![Livecycle Docker Extension](https://github.com/livecycle/preevy/assets/51878265/04e00790-beaf-4f22-bd6e-ca7f497f5aaa)
 
 That's it! You have successfully installed the Livecycle Docker Extension and shared your local development environment with your team.
 
+## Share to the cloud
+
+With the Livecycle Docker Extension, you can we can make your local environment accessible to your team members even when your machine is offline. This is done by putting your application to the cloud and creating a tunnel to it. This is useful when you want to share your environment with your team members for a longer period of time.
+
+### Deploy a new shareable environment to the cloud
+   
+   After you follow the steps in the [Getting Started](#Getting-Started) section to set up the Livecycle Docker Extension, instead of choosing the local option, choose the cloud option. After that, you will be prompted to choose to cloud provider. You can choose between AWS, GCP, and Azure. Or use Kubernetes if you have a Kubernetes cluster. Make sure you have the credentials for the cloud provider you choose and for Kubernetes you have the kubeconfig file. 
+
+  ![Livecycle Docker Extension](https://github.com/livecycle/preevy/assets/51878265/e44b5071-4014-4a6e-b59a-23cefee77908)
+
+  I will choose AWS as my cloud provider. As you can see it ask for region, Bundle id, etc. You can leave the region as default or choose the region you want. Also, some columns are optional, you can leave them blank. After that, click on the "Share" button to get the shareable URL.
```

Comment:
```suggestion
For this example, I will choose AWS as my cloud provider. As shown, it will ask for the region, Bundle ID, etc. You can leave the region as default or choose the region you prefer. Some fields are optional, so you can leave them blank. After that, click on the "Share" button to obtain the shareable URL.
```